Scripts/list_option_values.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+# Just leading spaces are remvoed and placeholders with square brackets.
```

Comment:
```suggestion
# Just leading spaces are removed and placeholders with square brackets.
```
Literally the only thing I can add to this conversation and it's not even a big deal

Reply:
It's fine. :) All suggestions are welcome. I will fix it tomorrow. 😊 Thanks.